src/screens/settings/css/profile.css

```diff
@@ -165,6 +223,28 @@ padding-left: 10px;
 font-weight: lighter;
 font-family: Arial, Helvetica, sans-serif;
 }
+#unique-url{
+margin-left: 11%;
+width: 70%;
+height: 40%;
+border-radius: 23px;
+border-color: lightgray;
+color: rgba(24, 17, 17, 0.871);
+padding-left: 10px;
+font-weight: bolder;
+font-family: Arial, Helvetica, sans-serif;
```

Comment:
the font family, seems different from the design, please can u recheck the figma design, thanks